x/oracle/spec/02_state.md

```diff
@@ -0,0 +1,61 @@
+<!--
+order: 2
+-->
+
+# State
+
+## ExchangeRate
+
+An `sdk.Dec` that stores an exchange rate against USD, which is used by the [Leverage](../../leverage/spec/README.md) module.
+
+- ExchangeRate: `0x01<denom_Bytes> -> amino(sdk.Dec)`
+
+## FeederDelegation
+
+An `sdk.AccAddress` (`umee-` account) address of `operator`'s delegated price feeder.
+
+- FeederDelegation: `0x02<valAddress_Bytes> -> amino(sdk.AccAddress)`
```

Comment:
We store this as proto bytes, not amino AFAIK.

Reply:
This applies to all state and thus this file 👍

Reply:
The relevant line in `x/oracle/keeper.go` is
```
store.Set(types.GetFeederDelegationKey(operator), delegatedFeeder.Bytes())
```
which stores the raw bytes. I don't see any proto or amino marshaling.

From `sdk.AccAddress.Bytes()`:
```go
// Bytes returns the raw address bytes.
func (aa AccAddress) Bytes() []byte {
	return aa
}
```